### PR TITLE
signup/register

### PR DESCRIPTION
**Pull request by DaudKhadijah01**

### CODE REVIEW

**Comment by DaudKhadijah01** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -25,16 +22,82 @@ textarea {
 
 #logo {
   float: left;
-  margin-right: 10px;
```
[the remark this answers is not in the document]
Comment (DaudKhadijah01) [reply app/assets/stylesheets/custom.scss:28]: image
I did not change anything.

**Comment by DaudKhadijah01** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -25,16 +22,82 @@ textarea {
...
-  letter-spacing: -1px;
-  padding-top: 9px;
-  font-weight: bold;
+    margin-right: 10px;
+    font-size: 1.7em;
+    color: #fff;
+    text-transform: uppercase;
+    letter-spacing: -1px;
+    padding-top: 9px;
+    font-weight: bold;
```
Comment (anlvq-1058) [inline app/assets/stylesheets/custom.scss:31]: wrong indent

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module UsersHelper
+
+  # Returns the Gravatar for the given user.
+  def gravatar_for(user)
+  gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+  gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+  image_tag(gravatar_url, alt: user.name, class: "gravatar")
```
Comment (anlvq-1058) [inline app/helpers/users_helper.rb:7]: indent please

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -10,10 +10,10 @@
   </head>
 
   <body>
-    <%= render "layouts/header" %>
+    <%= render 'layouts/header' %>
     <div class="container">
       <% flash.each do |message_type, message| %>
-        <div class="alert alert-<%= message_type %>"><%= message %></div>
+         <div class="alert alert-<%= message_type %>"><%= message %></div>
```
Comment (anlvq-1058) [inline app/views/layouts/application.html.erb:16]: wrong indent

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        <%= f.text_field :name %>
+        <%= f.label :email %>
+        <%= f.email_field :email %>
+        <%= f.label :password %>
+        <%= f.password_field :password %>
+        <%= f.label :password_confirmation, t("password_confirmation") %>
+        <%= f.password_field :password_confirmation %>
+        <%= f.submit "Register", class: "btn btn-primary" %>
+      <% end %>
+   </div>
```
Comment (anlvq-1058) [inline app/views/users/new.html.erb:31]: wrong indent

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by DaudKhadijah01** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**